docs/howto-openshift-crc.md

```diff
@@ -0,0 +1,260 @@
+(SCC) for its pods and containers. Before deploying the samba operator,
+the user should setup the `samba` SCC on the cluster. In order to
+deploy samba SCC manually, execute the following commands:
+
+```sh
+  $ cd samba-operator-dir
+  $ export KUBECTL_CMD=oc
+  $ $KUBECTL_CMD create -f config/openshift/scc.yaml
+  securitycontextconstraints.security.openshift.io/samba created
+  $KUBECTL_CMD get scc/samba -o yaml
+  ...
+```
+
+## Deploy samba-operator
+Enable developer mode and deploy the samba-operator using top-level
+Makefile target `make-deploy`. Wait for `samba-operator` pod to be in
+`Running` state:
+
+```sh
+  $ cd samba-operator-dir
+  $ export KUBECTL_CMD=oc
+  $ echo DEVELOPER=1 >> devel.mk
+  $ make deploy
+  ...
+  $ $KUBECTL_CMD get pods -n samba-operator-system
+  NAME                                                 READY   STATUS    RESTARTS   AGE
+  samba-operator-controller-manager-7c877459d4-wln54   2/2     Running   0          27s
+```
+
+## Create samba share
+Use the `smbtest.yaml` file below to simple SMB share. Wait for the share pod
+to be in `Running` state (may take some time):
+
+```sh
+  $ export KUBECTL_CMD=oc
+  $ $KUBECTL_CMD create -f smbtest.yaml
+  ...
+  $ $KUBECTL_CMD get pods -n smbtest
+  NAME                      READY   STATUS    RESTARTS   AGE
+  share1-5f7dbd45bc-bljrv   2/2     Running   0          4m23s
+```
+
+
+```yaml
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: smbtest
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: smb-pv
+  labels:
+    type: local
+spec:
+  storageClassName: manual
+  capacity:
+    storage: 8Gi
+  accessModes:
+    - ReadWriteOnce
+  hostPath:
+    path: "/mnt/export"
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: smb-pvc
+  namespace: smbtest
+spec:
+  storageClassName: manual
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 4Gi
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: users
+  namespace: smbtest
+type: Opaque
+stringData:
+  demousers: |
+    {
+      "samba-container-config": "v0",
+      "users": {
+        "all_entries": [
+          {
+            "name": "user1",
+            "password": "123456"
+          },
+          {
+            "name": "user2",
+            "password": "123456"
+          }
+        ]
+      }
+    }
+---
+apiVersion: samba-operator.samba.org/v1alpha1
+kind: SmbSecurityConfig
+metadata:
+  name: users
+  namespace: smbtest
+spec:
+  mode: user
+  users:
+    secret: users
+    key: demousers
+---
+apiVersion: samba-operator.samba.org/v1alpha1
+kind: SmbCommonConfig
+metadata:
+  name: config
+  namespace: smbtest
+spec:
+  network:
+    publish: cluster
+---
+apiVersion: samba-operator.samba.org/v1alpha1
+kind: SmbShare
+metadata:
+  name: share1
+  namespace: smbtest
+spec:
+  securityConfig: users
+  readOnly: false
+  storage:
+    pvc:
+      name: "smb-pvc"
+...
```

Comment:
This section is meant to be copy-n-pasted. Thus we shouldn't put an elipsis  here as this is not valid syntax.